report_dynamic/models/header.py

```diff
@@ -0,0 +1,27 @@
+class Header:
+    def __init__(self, child=False, parent=False):
+        self.value = 0
+        self.base_value = 0
+        self.child = child
+        if parent:
+            parent.child = self
+
+    @property
+    def next(self):
+        self.value += 1
+        if self.child:
+            self.child.reset  # pylint: disable=pointless-statement
```

Comment:
```suggestion
            self.child.reset()
```
wouldn't be better to make `reset` a method?